LegionCalculator/src/hyper.js

```diff
@@ -2420,7 +2445,8 @@ function getClassData(maple_class) {
             'attPercent': 4,
             'iedPercent': [44, 20],
             'dmgPercent': 20,
-            'bossPercent': 0
+            'bossPercent': 0,
+            'critDmg': 0
```

Comment:
Forgot to apply this to the hyper stat optimizer as well. I can also extract this fix to a separate pull request if you'd like.